ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift

```diff
@@ -108,27 +112,50 @@ struct TimelineItemBubbledStylerView<Content: View, DeliveryStatus: View>: View
                              cornerRadius: cornerRadius,
                              corners: roundedCorners)
         } else {
-            VStack(alignment: .trailing, spacing: 4) {
-                contentWithReply
+            contentWithTimestamp
+                .bubbleStyle(inset: true,
+                             color: timelineItem.isOutgoing ? .element.bubblesYou : .element.bubblesNotYou,
+                             cornerRadius: cornerRadius,
+                             corners: roundedCorners)
+        }
+    }
 
-                if timelineItem.properties.isEdited {
-                    Text(L10n.commonEditedSuffix)
-                        .font(.compound.bodyXS)
-                        .foregroundColor(.element.tertiaryContent)
+    @ViewBuilder
+    var contentWithTimestamp: some View {
+        if isTextItem {
+            ZStack(alignment: .topLeading) {
+                contentWithReply
+                    .layoutPriority(1)
+                VStack(alignment: .trailing, spacing: 0) {
+                    Spacer()
+                    HStack(alignment: .bottom, spacing: 0) {
+                        Spacer()
+                        timestamp
+                    }
                 }
```

Comment:
Probably a bit simpler to do it without the stacks & spacers:
```suggestion
                timestamp
                    .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .bottomTrailing)
```